src/App.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <el-container id="app" direction="vertical">
     <Navbar />
-    <el-main>
+    <div>
```

Comment:
@weiweitoo what's the reason for removing el-main? Was it getting in the way of something?